cloudinit/sources/azure/kvp.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2022 Microsoft Corporation.
+#
+# This file is part of cloud-init. See LICENSE file for license information.
+
+import logging
+from typing import Optional
+
+from cloudinit.reporting import handlers, instantiated_handler_registry
+from cloudinit.sources.azure import errors
+
+LOG = logging.getLogger(__name__)
+
+
+def get_kvp_handler() -> Optional[handlers.HyperVKvpReportingHandler]:
+    """Get instantiated KVP telemetry handler."""
+    kvp_handler = instantiated_handler_registry.registered_items.get(
+        "telemetry"
+    )
+    if not isinstance(kvp_handler, handlers.HyperVKvpReportingHandler):
+        LOG.debug("KVP handler not enabled, skipping host report.")
```

Comment:
Remove this debug print. Caller of this method can decice whether to print any debug or not. Currently report_via_kvp will also print a message if kvp is not enabled which means the same message is printed twice

Reply:
whoops, that was not meant to be there. thanks!

Reply:
fixed